### PR TITLE
Clone from https, build on Mac

### PR DESCRIPTION
**Pull request by behoyh**
Small change to prep-img.sh that simplifies rsa key management (in the case it is not already configured with GitHub)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+WORKDIR /root/build
+  
+RUN  apt-get update \
+  && apt-get install -y wget unzip whois git tree sudo openssh-client \
+  && rm -rf /var/lib/apt/lists/*
+
+ADD ./build /root/build
+ADD ./scripts /root/scripts
+
+CMD   ssh-keygen \
```
Comment (pmansour) [inline Dockerfile:16]: this would leave your newly-generated SSH creds on the build container, so they may not be accessible after you actually build this. Would it be better to try and find / mount any existing SSH creds to the build container, rather than generating new ones?

File: build/prep-img.sh [hunk 2]
```diff
@@ -144,7 +144,7 @@ debug 'config.txt' "`cat "$BOOT_CONFIG_FILE" | grep 'gpio-fan'`"
 echo
 echo "Copying programs.."
 GIT_TMP=`mktemp -d`
-git clone -q git@github.com:pmansour/minimeet.git "$GIT_TMP/minimeet/"
+git clone -q https://github.com/pmansour/minimeet.git "$GIT_TMP/minimeet/"
```
Comment (pmansour) [inline build/prep-img.sh:147]: this is a great change useful across all OS versions. Do you think you could separate this into its own PR so we can merge it straight away? The other changes might need some more thinking.

File: build/write-img-mac.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+# Prerequisites: Docker
+# Tested on MacOS 12.2.1
+# Author: behoyh
+
+# NOTE: EXPERIMENTAL. Use at your own risk!
+# Requires Dockerfile to have written .iso image first, eg.
+# docker run -it -v /dev:/dev -v $HOME/raspios-img/output/:/root/raspios-img/ --privileged pimeet
+# where '$HOME/raspios-img/output/' is the location of the image on the host.
+
+IMG_FILE="$HOME/raspios-img/output/2022-01-28-raspios-bullseye-arm64.img"
```
Comment (pmansour) [inline build/write-img-mac.sh:11]: FYI I updated this file name to avoid having to change all the scripts whenever we upgrade OS version. Take a look at the recent scripts to see the new name.

File: build/write-img-mac.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (pmansour) [inline build/write-img-mac.sh:1]: I'm torn on whether having a separate fork of this file for mac would be useful / maintainable. Aside from `write-img`, do you know if the other scripts even work on mac right now?

**Review by pmansour — COMMENTED** [review]
thanks Beshoy!

File: Dockerfile (hunk 1 above)
Comment (behoyh) [reply Dockerfile:16]: that will work as well but may be under a different user scope.

File: build/prep-img.sh (hunk 2 above)
Comment (behoyh) [reply build/prep-img.sh:147]: New PR raised

File: build/write-img-mac.sh (hunk 4 above)
Comment (behoyh) [reply build/write-img-mac.sh:1]: download-image works as I recall.

File: Dockerfile (hunk 1 above)
Comment (behoyh) [reply Dockerfile:16]: I'm honestly not too sure, since it's been awhile since I've used this particular image.
